docs/UserGuide.md

```diff
@@ -223,30 +228,37 @@ Deletes the specified contact from the **Contacts** tab using the keyword. Will
 
 Format: `delete KEYWORD [MORE_KEYWORDS]`
 
+<box type="warning" seamless>
+
 * Delete contact(s) that contains `KEYWORD`
+
 * Able to delete multiple contacts at once containing `KEYWORD`(s)
+
 * Using partial keywords will be matched. e.g. ha will match hans
 > `ha` → 3. Hans Gruber
+
 * The search is case-insensitive. e.g. `hAnS` will match `Hans`
 > `hAnS` → 4. Hans Gruber
+
 * Persons matching at least one keyword will be returned (i.e. OR search). e.g. Hans Bo will return Hans Gruber, Bo Yang
 > `Hans Bo` → 3. Hans Gruber
 >             4. Bo Yang 
+
 * The order of the keywords does not matter. e.g. Hans Bo will match Bo Hans
-> `Bo Hans` → 3. Hans Gruber
-> 
->             4. Bo Yang
+> `Bo Hans` → 3. Hans Gruber `/` 4. Bo Yang
+
 * Only the `NAME` of the contact is searched
```

Comment:
Is there a difference between this example and the one above it on lines 243 - 245? Is there a reason to have a '/' character in one and not the other?

Reply:
ok i have changed it to only show the example once and the next point to refer to the result above